bitwarden_license/bit-web/src/app/secrets-manager/shared/projects-list.component.ts

```diff
@@ -89,4 +92,17 @@ export class ProjectsListComponent {
     }
     return false;
   }
+
+  static copyProjectUuid(
```

Comment:
Making it explicit on intention is better, by adding `clipboard` to name
```suggestion
  static copyProjectUuidToClipboard(
```